scrappers/tbib/lib/Tbib.rb

```diff
@@ -0,0 +1,76 @@
+class Tbib
+	include RethinkDB::Shortcuts
+
+	def initialize()
+		@conn = r.connect(:host => $config.server_ip, :port => $config.server_port)
+		$log.log 'connected to database!'
+		@conn.use($config.database)
+		@mutex = Mutex.new
+		if !r.table_list.run(@conn).include?($config.table)
+			r.table_create($config.table).run(@conn)
```

Comment:
The table should use the image `id` as primary key, you can read how to do that [here](https://www.rethinkdb.com/api/ruby/table_create/)

Reply:
done ;)
`r.table_create($config.table, :primary_key => 'id').run(@conn)`